tests/test_core_schedule.py

```diff
@@ -1295,6 +1295,23 @@ def test_generating_new_vehicles_with_overwite_True(schedule):
     assert_semantically_equal(schedule.vehicles, {'veh_3_bus': {'type': 'bus'}, 'veh_4_bus': {'type': 'bus'},
                                                   'veh_1_bus': {'type': 'bus'}, 'veh_2_bus': {'type': 'bus'}})
 
+def test_scale_vehicle_capacity(schedule, tmpdir):
+    
+    schedule = read.read_matsim_schedule(
+    path_to_schedule=pt2matsim_schedule_file,
+    path_to_vehicles=pt2matsim_vehicles_file,
+    epsg='epsg:27700')
```

Comment:
you dont need to read in a schedule, the test function takes a schedule object you can use - I think it might actually be the same schedule you're reading in here :)